qiskit_experiments/data_processing/sklearn_discriminators.py

```diff
@@ -99,3 +103,72 @@ def from_config(cls, config: Dict[str, Any]) -> "SkLDA":
                 setattr(lda, name, value)
 
         return SkLDA(lda)
+
+
+class SkCLF(BaseDiscriminator):
+    """A wrapper for the SKlearn classfier Pipeline."""
+
+    def __init__(self, sgdc: "SGDClassifier"):

```

Comment:
will change this to `ClassifierMixin` because that's the base class of the stochastic classifiers and the discriminants. This way it can also accept an e.g. `QuadraticDiscriminantAnalysis`.

Reply:
Addressed in commit [9390381](https://github.com/Qiskit/qiskit-experiments/pull/936/commits/9390381ce44f62e19b8243b76bd212b37df0fa8f).